using-virtual-machines/guest-agent.adoc

```diff
@@ -0,0 +1,161 @@
+Guest Agent information from the Virtual Machine
+------------------------------------------------
+
+Guest Agent (GA) is an optional component running inside of the Virtual Machine.
+The GA provides plenty of additional runtime information about the running operating system (OS).
+More technical detail about available GA commands is in the https://qemu.weilnetz.de/doc/3.1/qemu-ga-ref.html[documentation].
+
+
+Guest Agent info in Virtual Machine status
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+GA presence in the Virtual Machine is signaled with a condition in the VirtualMachineInstance status.
+The condition tells that the GA is connected and can be used.
+
+.GA condition
+[source,yaml]
+----
+status:
+    conditions:
+        - lastProbeTime: "2020-02-28T10:22:59Z"
+          lastTransitionTime: null
+          status: "True"
+          type: AgentConnected
+----
+
+When the GA is connected, additional OS information is shown in the status.
+This information comprises:
+
+   - guest info, which contains OS runtime data
+   - interfaces info, which shows QEMU interfaces merged with GA interfaces info.
+
+Bellow is the example of the information shown in the VirtualMachineInstance status.
+
+.GA info merged into status
+[source,yaml]
+----
+status:
+    guestOSInfo:
+        id: fedora
+        kernelRelease: 4.18.16-300.fc29.x86_64
+        kernelVersion: '#1 SMP Sat Oct 20 23:24:08 UTC 2018'
+        name: Fedora
+        prettyName: Fedora 29 (Cloud Edition)
+        version: "29"
+        versionId: "29"
+    interfaces:
+        - interfaceName: eth0
+          ipAddress: 10.244.0.23/24
+          ipAddresses:
+              - 10.244.0.23/24
+              - fe80::858:aff:fef4:17/64
+          mac: 0a:58:0a:f4:00:17
+          name: default
+----
+
+When the GA is not present in the Virtual Machine, the information is not shown, and no error is given.
```

Comment:
When the Guest Agent is not present in the Virtual Machine, the information is not shown. No error is reported because the Guest Agent is an optional component.